test/irmin-unix/test_command_line.t

```diff
@@ -86,7 +86,7 @@ Check mismatched hash function
   $ irmin set --root ./test-hash -s irf -h sha1 abc 123
```

Comment:
For the above line, perhaps change to: "Check mismatched hash function. NOTE the format of the error message changes when ocamlformat is installed in the switch; the following assumes ocamlformat IS installed."